RELEASE-NOTES.md

```diff
@@ -1,5 +1,19 @@
 # Prefect Release Notes
 
+## Release 2.14.1
+
+### Documentation
+- Add Python `serve` and `deploy` options to the `schedules` concepts documentation — https://github.com/PrefectHQ/prefect/pull/11000
+
+### Fixes
+- Refine flow parameter validation to use the correct form of validation depending on if the parameter is a pydantic v1 or v2 model.  — https://github.com/PrefectHQ/prefect/pull/11028
+
+### Contributors
+- @chrisguidry
```

Comment:
I think we usually exclude Prefect contributors?

Reply:
I always forget that - thanks for catching.